pkg/block/s3/adapter.go

```diff
@@ -741,6 +786,9 @@ func (a *Adapter) GetStorageNamespaceInfo() block.StorageNamespaceInfo {
 	if !(a.disablePreSignedUI || a.disablePreSigned) {
 		info.PreSignSupportUI = true
 	}
+	if !a.disablePreSignedMultipart || !info.PreSignSupport {
+		info.PreSignSupportMultipart = true
+	}
```

Comment:
I don't understand. If `PreSignSupport` is false, `PreSignSupportMultipart` is true? why?

Reply:
So shouldn't it be?
```suggestion
	if !a.disablePreSignedMultipart && info.PreSignSupport {
		info.PreSignSupportMultipart = true
	}
```

Reply:
Fixed my bad